integration_tests/jira_issues_tests.feature

```diff
@@ -52,6 +52,7 @@ Feature: Jira Issues Tests
      When we run the simulation with setup file jira.d3asim_871 and parameters [24, 60, 60, 1]
      Then there should be no unmatched loads
 
+  @disabled
```

Comment:
Sorry for nitpicking, and it is really not that important since we probably will not need alternative pricing in the future, however why was this test disabled ?

Reply:
@spyrostz There is a problem with the alternative pricing in the storage strategy in the one-sided market.
This is why i disabled this integrationtests: https://github.com/gridsingularity/d3a/blob/b49516f7ba99c9c4ca0f8b185d063698cb40f3f1/integration_tests/alternative_pricing.feature#L9 already some weeks ago.
I tried to fix it but gave up after some time because as you said: probably not needed any more.
The changes i did in the storage strategy originate from the try of fixing.

Reply:
Thanks for clarifying this, it indeed looks like a waste of time and we should fix it if we plan to revamp alternative pricing.